modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/Rails5ServerCodegen.java

```diff
@@ -281,8 +281,8 @@ public String toApiFilename(String name) {
         // replace - with _ e.g. created-at => created_at
         name = name.replaceAll("-", "_"); // FIXME: a parameter should not be assigned. Also declare the methods parameters as 'final'.
 
-        // e.g. PhoneNumberApi.rb => phone_number_api.rb
-        return underscore(name) + "_controllers";
+        // e.g. DefaultController => defaults_controller.rb
+        return underscore(name) + "s_controller";
```

Comment:
This is incorrect and breaks. The filename and the class name should match. It'd be better to make them whatever the tag is so your explicitly setting it to what you want. Controllers don't have inflections like models.